recommendations/DP0003-laws-of-identity/DP0003-laws-of-identity.md

```diff
@@ -10,7 +10,8 @@
 
 
 ## Summary
-The **Seven Laws of Identity** were [published by Kim Cameron, Architect of Identity at the Microsoft Corporation](https://www.identityblog.com/stories/2005/05/13/TheLawsOfIdentity.pdf) in 2005. This document serves to recognize this work as a collection of design principles for identity solutions.  
```

Comment:
In who's work and how are these presido principles relevant to laws of identity.

Reply:
Frankly speaking, I didn't plan to add Presidio Principles to this document. I have followed on your proposal and added them. I would suggest to leave this document without them. Especially, since we have added Presidio Principles to Governance Stack Design Principles.

Reply:
+1